.github/workflows/dependency-tests.yml

```diff
@@ -49,7 +44,10 @@ jobs:
         platform: [ubuntu-latest, macos-latest, windows-latest]
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: lf-lang/vscode-lingua-franca/.github/actions/build@main
+      - uses: lf-lang/vscode-lingua-franca/.github/actions/build@update-error-message
```

Comment:
If we check out the repo in this workflow instead of `build.yml`, I think we could refer to it with a local path and won't need the `@`.

Reply:
I tried that and it did not work conveniently, but I do not remember why. It could just be that I would need to spend a bit of time rearranging some YAML, but I have not found the time to do that yet.

Reply:
OK, then let's not waste time on it. It's not a big deal. Let's merge, adjust the refs, and only then delete this branch.